packages/nhsuk-frontend/src/nhsuk/components/header/macro-options.mjs

```diff
@@ -276,85 +276,108 @@ export const params = {
   }
 }
 
+const nhsukNavigationItems = [
```

Comment:
Lovely stuff here @paulrobertlloyd!

Well worth the hard work 🙌

Do you mind keeping these files as dumb as possible and avoid this const?

Repetitive yeah, but easy to edit for all

Reply:
Done.